roles/openshift_cluster_monitoring_operator/defaults/main.yml

```diff
@@ -6,6 +6,8 @@ openshift_cluster_monitoring_operator_alertmanager_repo: quay.io/prometheus/aler
 openshift_cluster_monitoring_operator_prometheus_reloader_repo: quay.io/coreos/prometheus-config-reloader
 openshift_cluster_monitoring_oeprator_configmap_reloader_repo: quay.io/coreos/configmap-reload
 
+openshift_cluster_monitoring_operator_cluster_id: "{{ openshift_master_cluster_public_hostname }}"
```

Comment:
@sdodson thinks this may not always be set; maybe `openshift_master_cluster_hostname` instead?

Reply:
WIP'ing this until I get a bit of feedback on the default.

Reply:
Implemented